src/node_env_var.cc

```diff
@@ -57,6 +58,8 @@ class MapKVStore final : public KVStore {
 namespace per_process {
 Mutex env_var_mutex;
 std::shared_ptr<KVStore> system_environment = std::make_shared<RealEnvStore>();
+std::shared_ptr<EnabledDebugList> enabled_debug_list =
+    std::make_shared<EnabledDebugList>(nullptr);
```

Comment:
Ditto here, is there any reason this needs to be a pointer? I don’t really see any, especially if this is kept alive for the duration of the process from here anyway